src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
```

Comment:
Consider using a constant for the price per day to avoid hardcoding. This will make your code more readable and maintainable. Refer to checklist item 2.